paddle/fluid/train/imdb_demo/train_filelist.txt

```diff
@@ -0,0 +1,12 @@
+train_data/part-0
```

Comment:
I think this file can be generated through shell scripts.

Reply:
train_filelist.txt can be very flexible, I decide to leave this part of the work to users.

Reply:
train_filelist.txt can be very flexible, I decide to leave this part of the work to users.